packages/contracts-bedrock/test/L2/SuperchainWETH.t.sol

```diff
@@ -45,6 +52,7 @@ contract SuperchainWETH_Test is CommonTest {
 
         // Arrange
         vm.deal(alice, _amount);
+        _mockAndExpect(address(l1Block), abi.encodeCall(l1Block.isCustomGasToken, ()), abi.encode(false));
```

Comment:
Not a strong opinion here, but at unit tests we are used to mock and expect calls to external dependencies so that we isolate the contract being tested. If we want to make more like an integration test what we can do here is to make two different test setups using the `CommonTest.setUp()` to override the custom gas token.

Reply:
Ok, let's leave this as-is for now since it's not a devnet blocker, but prior to mainnet we should consider how to strengthen up the test suite by removing mocks. I'll defer to you on where to track that task :)